Source/DataSources/KmlDataSource.js

```diff
@@ -489,8 +494,8 @@ define([
         }
         if (!hrefResolved && defined(sourceUri)) {
             href = getAbsoluteUri(href, getAbsoluteUri(sourceUri));
-            href = proxyUrl(href, proxy);
         }
+        href = proxyUrl(href, proxy, query);
```

Comment:
This still needs to be inside of an `if (!hrefResolved)` block, so adjust the original logic to handle that.  Basically, if `hrefResolved` is true at this point, it means that the item is a data or blob uri, which means we can't/shouldn't append query parameters to it.